src/Slider/Slider.tsx

```diff
@@ -50,34 +56,65 @@ const Slider: React.FunctionComponent<SliderProps> = (props: SliderProps): React
             });
             setLabelsPositions(positions);
         }
-    }, [props.labels]);
+    }, [props.labels, min, max]);
+
+    React.useEffect(() => setThumbPosition(getPercentage()), [props.value, min, max, size]);
+
+    /**
+     * Finds the size between two numbers
+     * @param {number} minValue The minimum value
+     * @param {number} maxValue The maximum value
+     * @returns {number} The size
+     */
+    function getSize(minValue: number, maxValue: number): number {
+        if (maxValue > minValue) {
+            return maxValue - minValue;
+        } else { // Will calculate the size anyway, but it will show a warning since the min is larger than the max
+            console.warn(`The max value of the slider should be larger than the min value (Max:${max}, Min: ${min}`);
```

Comment:
This warning is not enough since it appears only on the console, we should move the warning to the error prop or make sure that the component doesnt accept min > max

Reply:
The error prop is controlled by the parent component so we can't use that. I only put a warning because I thought this should be something that is caught during development. The calculation will proceed with no issue but it shouldn't happen. The only scenario I can think of is when someone mistakenly thinks that the slider could go backward if it's in negative like setting the `min` to `0` and the `max` to `-30` thinking that it's a `from` - `to`. In this case, this warning will be shown.